src/templates/legal/regulation.html.tt

```diff
@@ -0,0 +1,198 @@
+<div class="static_full">
+    <h1 class="center-text">[% l("Regulatory Information") %]</h1>
+    <div class="gr-parent gr-padding-30">
+        <p class="center-text">
+            [% l("The Binary.com service is owned and operated by Binary Ltd. Fully licensed and regulated, Binary.com has been providing financial trading to a worldwide audience of over 1,000,000 accounts since 1999.") %]
```

Comment:
actually please remove the sentence "The Binary.com service is owned and operated by Binary Ltd.", because it isn't technically true: the website is operated by the subsidiaries listed on this page, not by Binary Ltd. itself.